azure-pipelines.yml

```diff
@@ -2,79 +2,87 @@
 # Create and test a Python package on multiple Python versions.
 # Add steps that analyze code, save the dist with the build record, publish to a PyPI-compatible index, and more:
 # https://docs.microsoft.com/azure/devops/pipelines/languages/python
-
 trigger:
 - master
 
 jobs:
 - job: Linux
   pool:
-    vmImage: 'ubuntu-16.04'
+    vmImage: 'ubuntu-20.04'
   strategy:
     matrix:
       Python36:
         python.version: '3.6'
       Python37:
         python.version: '3.7'
       Python38:
-        python.version: '3.8'
+        python.version: '3.7'
```

Comment:
Is there a reason the Python38 job is using 3.7, or is this a typo?

Reply:
definitely a typo